### DISCUSSION TITLE
Model / Ensemble Recommendations For:  Podcast / Conversation / Spoken Word

### DISCUSSION THREAD

**Discussion by phatlix** [Anjok07/ultimatevocalremovergui#732] [Q&A]
Hello!  

I am looking for a model and/or an ensemble collection that could isolate only vocals cleanly, while removing all background noise. Background noise might include light chatter from other people in the distance, or a low volume tv through walls.  

I'm looking to just isolate a conversation.  

UVR is absolutely fantastic! I have been trying different models and combinations, but I'm having trouble nailing down the perfect recipe.  Working with music has just been incredible!  So I am super hopeful that UVR could knock spoken word out of the park as well.  

If others have used UVR for doing something similar with decent results, I would be very interested and very much appreciative if you could share your findings.  

Thank you!

**Comment by dgoryeo** [comment]
Hi @phatlix , did you find a solution to your question? I'm looking for similar setup. Thanks
